core/src/main/resources/error/error-classes.json

```diff
@@ -888,6 +888,11 @@
         "message" : [
           "IN/EXISTS predicate subqueries can only be used in filters, joins, aggregations, window functions, projections, and UPDATE/MERGE/DELETE commands<treeNode>"
         ]
+      },
+      "UNSUPPORTED_OUTER_REFERENCE_DATA_TYPE" : {
+        "message" : [
+          "Correlated column references do not support data type <dataType>: <expr>"
```

Comment:
Nit: I found it was a bit confusing to read the error message instance: `Correlated column references do not support data type map<string,int>: v1.x`. 

Basically  `v1.x` as an expression was not clear in the error message.

For example to me this is more clear: `Correlated column references do not support expression v1.x due to unsupported data type map<string,int>`. Something like this.

Reply:
Maybe just say:
```
The data type of correlated column references cannot be/contain map type.
```

Reply:
Thanks @amaliujia @cloud-fan I will update the error message.